timescaledb/how-to-guides/connecting/azure-data-studio.md

```diff
@@ -18,29 +18,19 @@ service.
 *   Download and install [Azure Data Studio][ms-azure-data-studio].
 *   Install the [PostgreSQL extension for Azure Data Studio][postgresql-azure-data-studio].
 
-## Connecting to TimescaleDB hosted locally, or on a remote server
+### Connecting to TimescaleDB hosted locally, or on a remote server
```

Comment:
I think this should stay an `h3`, because it's not a subheading of `Before you begin`